mavenproject/src/main/java/mtm/mavenproject/Sorcery.java

```diff
@@ -0,0 +1,21 @@
+package mtm.mavenproject;
+
+public class Sorcery extends Card{
+
+	public Sorcery(String name, int gemCost, String gemColor) {
+		super(name, gemCost, gemColor);
+		// TODO Auto-generated constructor stub
+	}
+	
+	public Sorcery(String name, int gemCost, String gemColor, String effect) {
+		super(name, gemCost, gemColor, effect);
+		// TODO Auto-generated constructor stub
+	}
+
+	@Override
+	public void Message() {
```

Comment:
This name should be lowercase. Also, I don't think any class should print to stdout, except when debugging. It would be better to have them return a string representation: the `toString()` method might be a good place for that.